<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>731</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <property name="sizeIncrement" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>ReZound</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>../../../../.designer/images/icon_logo_16.png</normaloff>../../../../.designer/images/icon_logo_16.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>71</y>
     <width>780</width>
     <height>555</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <property name="spacing" >
     <number>0</number>
    </property>
    <property name="margin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="contentFrame" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>708</y>
     <width>780</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="transportDockWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>26</y>
     <width>256</width>
     <height>39</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Transport</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>23</y>
      <width>256</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="TransportWindow" native="1" name="m_transportWindow" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="miscDockWidget" >
   <property name="geometry" >
    <rect>
     <x>262</x>
     <y>26</y>
     <width>256</width>
     <height>39</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Misc Controls</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>23</y>
      <width>256</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="MiscControlsWindow" native="1" name="m_miscControlsWindow" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="soundListDockWidget" >
   <property name="geometry" >
    <rect>
     <x>524</x>
     <y>26</y>
     <width>256</width>
     <height>39</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Loaded Sounds</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>23</y>
      <width>256</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="SoundListWindow" native="1" name="m_soundListWindow" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="levelMetersDockWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>632</y>
     <width>256</width>
     <height>76</height>
    </rect>
   </property>
   <property name="features" >
    <set>QDockWidget::DockWidgetFeatureMask</set>
   </property>
   <property name="allowedAreas" >
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Levels</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <property name="geometry" >
     <rect>
      <x>23</x>
      <y>0</y>
      <width>233</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="PlaybackLevelMeters" native="1" name="m_levelMeters" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="stereoPhaseMetersDockWidget" >
   <property name="geometry" >
    <rect>
     <x>262</x>
     <y>632</y>
     <width>256</width>
     <height>76</height>
    </rect>
   </property>
   <property name="features" >
    <set>QDockWidget::DockWidgetFeatureMask</set>
   </property>
   <property name="windowTitle" >
    <string>Stereo Phase</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2" >
    <property name="geometry" >
     <rect>
      <x>23</x>
      <y>0</y>
      <width>233</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="StereoPhaseMeters" native="1" name="m_stereoPhaseMeters" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="frequencyAnalyzerDockWidget" >
   <property name="geometry" >
    <rect>
     <x>524</x>
     <y>632</y>
     <width>256</width>
     <height>76</height>
    </rect>
   </property>
   <property name="features" >
    <set>QDockWidget::DockWidgetFeatureMask</set>
   </property>
   <property name="windowTitle" >
    <string>Frequency Analyzer</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7" >
    <property name="geometry" >
     <rect>
      <x>23</x>
      <y>0</y>
      <width>233</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="FrequencyAnalyzer" native="1" name="m_frequencyAnalyzer" />
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Open" >
   <property name="text" >
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Quit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionTest1" >
   <property name="text" >
    <string>test1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="4" margin="4" />
 <customwidgets>
  <customwidget>
   <class>TransportWindow</class>
   <extends>QWidget</extends>
   <header>TransportWindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MiscControlsWindow</class>
   <extends>QWidget</extends>
   <header>MiscControlsWindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SoundListWindow</class>
   <extends>QWidget</extends>
   <header>SoundListWindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlaybackLevelMeters</class>
   <extends>QWidget</extends>
   <header>LevelMeters.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StereoPhaseMeters</class>
   <extends>QWidget</extends>
   <header>StereoPhaseMeters.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FrequencyAnalyzer</class>
   <extends>QWidget</extends>
   <header>FrequencyAnalyzer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
